src/main/osd/osd_warnings.c

```diff
@@ -177,6 +177,15 @@ void renderOsdWarning(char *warningText, bool *blinking, uint8_t *displayAttr)
         return;
     }
 #endif // USE_RX_RSSI_DBM
+#ifdef USE_RX_RSNR
+    // rsnr
+    if (osdWarnGetState(OSD_WARNING_RSNR) && (getRsnr() < osdConfig()->rsnr_alarm)) {
+        tfp_sprintf(warningText, "RSNR");
+        *displayAttr = DISPLAYPORT_ATTR_WARNING;
+        *blinking = true;;
```

Comment:
double semicolon

Reply:
Oh, I see this comes from where I copy pasted, and it's widespread not only in this file but also in other files in the project, so I guess it's time to fix them all.